### ISSUE TITLE
Different pets with the same name all deleted

### ISSUE THREAD

**Issue by PatrykBochenek** [tonybaloney/vscode-pets#386]
In a scenario where there are two or more pets with the same name, upon deleting one of them, all of the pets with the same name are deleted. 

<img width="299" alt="image" src="https://user-images.githubusercontent.com/106195747/217894885-287a76a8-2590-4272-8024-704f8fa109ae.png">
 
For example, as pictured above if I delete a pet called 'test' then it will cause all the other pets called 'test' to be deleted too.

**Comment by AndreasWintherMoen** [comment]
Yeah, using name as ID isn't really ideal. Should probably use [uuid](https://www.npmjs.com/package/uuid) instead to uniquely identify pets.
